<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataCode</class>
 <widget class="QWidget" name="DataCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataCode</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(238, 255, 250, 200);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>通过</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>失败</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>加密测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;测试步骤:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;上位机通过 usb 读取数据加密 ic I2C 地址；&lt;/p&gt;&lt;p&gt;i. 根据弹出框中读取到的id，点击通过或失败按钮。&lt;/p&gt;&lt;p&gt;ii. 测试加密功能是否ok。&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;判定：&lt;/span&gt;&lt;/p&gt;&lt;p&gt;读取的 I2C 地址与预设的地址一致则自动判定为 Pass；否则为Fail。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
image: url(:/ProductTest_360/Tip.jpg);
border-style: outset;
border-width: 4px;
border-radius: 10px;
border-color: rgba(255, 170, 0, 30);
}
QLabel:hover{
background-color:rgba(100,255,100, 100);
border-color: rgba(255, 170, 0, 255);
color:rgba(0, 0, 0, 255);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ProductTest_360.qrc"/>
 </resources>
 <connections/>
</ui>
